docs/multicluster/user-guide.md

```diff
@@ -387,29 +394,36 @@ internalIP: 10.17.27.55
-need to make sure the resulted `gatewayIP` can be reached from the remote
-Gateways.
-
-After the `Gateway` CR is created, Multi-cluster Controller will be responsible
-for exporting the cluster's network information to the leader cluster including
-the Gateway IPs and `serviceCIDR` (the cluster's Service ClusterIP range).
-Multi-cluster Controller will try to discover `serviceCIDR` automatically, but
-you can also define the `serviceCIDR` manually in the Antrea Multi-cluster
-ConfigMap `antrea-mc-controller-config-***`.
-
-The Multi-cluster resource export/import pipeline will replicate the exported
-cluster network information to all member clusters in the ClusterSet. For
-example, in other member clusters, you can see a `ClusterInfoImport` CR with
-name `test-cluster-east-clusterinfo` is created for `test-cluster-east` with
-its network information. You can check the `ClusterInfoImport` with command:
-`kubectl get clusterinfoimport test-cluster-east-clusterinfo -o yaml`, which
-should show information like:
+remote Gateways of other member clusters. Multi-cluster Controller discovers the
+IP addresses from the K8s Node resource of the Gateway Node. It will always use
+`InternalIP` of the K8s Node as the Gateway's `internalIP`. For `gatewayIP`,
+there are several possibilities:
+
+* By default, the K8s Node's `InternalIP` is used as `gatewayIP` too.
+* You can choose to use the K8s Node's `ExternalIP` as `gatewayIP`, by changing
+the configuration option `gatewayIPPrecedence` to value: `public`, when
+deploying the member Multi-cluster Controller. The configration option is
+defined in ConfigMap `antrea-mc-controller-config-***` in `antrea-multicluster-member.yml`.
+* When the Gateway Node has a separate IP for external communication or is
+associated with a public IP (e.g. an Elastic IP on AWS), but the IP is not added
+to the K8s Node, you can still choose to use the IP as `gatewayIP`, by adding an
+annotation: `multicluster.antrea.io/gateway-ip=<ip-address>` to the K8s Node.
+
+When selecting the Multi-cluster Gateway Node, you need to make sure the
+resulted `gatewayIP` can be reached from the remote Gateways.
+
+After the Gateway is detected, Multi-cluster Controller will be responsible
+for exporting the cluster's network information to other member clusters
+through the leader cluster, including the cluster's Gateway IP and Service
+CIDR. Multi-cluster Controller will try to discover the cluster's Service CIDR
+automatically, but you can also manually specify the `serviceCIDR` option in
+ConfigMap `antrea-mc-controller-config-***`. In other member clusters, a
+`ClusterInfoImport` CR will be created for the cluster which includes the
+exported network information. For example, in cluster `test-cluster-west`, you
+you can see a `ClusterInfoImport` CR with name `test-cluster-east-clusterinfo`
+is created for cluster `test-cluster-east`. You can check the
+`ClusterInfoImport` with command: `kubectl get clusterinfoimport -o yaml`,
```

Comment:
```suggestion
`ClusterInfoImport` with command: `kubectl get clusterinfoimport test-cluster-east-clusterinfo -o yaml`,
```

Reply:
I feel no need to specify name here. Anyway readers can understand they can add name if they want. In most cases, they will not, to list all clusters.